broker/setup_broker/lvk/setup_broker.sh

```diff
@@ -8,7 +8,9 @@ teardown="${2:-False}"
 # "True" tearsdown/deletes resources, else setup
 survey="${3:-lvk}"
 # name of the survey this broker instance will ingest
-region="${4:-us-central1}"
+observation_run="${4:-4}"
```

Comment:
Leave this variable named `schema_version` like we do for other surveys. That's what this variable really is and how it's used in the code. This will need to be changed in more places as well.

```suggestion
schema_version="${4:-4}"
```